### PR TITLE
Created button component with a circular theme

### PR DESCRIPTION
**Pull request by Siffster85**

### CODE REVIEW

**Review by Karumu05 — APPROVED** [review]

File: app/(tabs)/profile/index.tsx [hunk 1]
```diff
@@ -1,10 +1,37 @@
 import { View, Text } from "@/components/Themed"
 import { Stack } from "expo-router";
+import * as ImagePicker from 'expo-image-picker';
+import { useState } from 'react';
+import Button from '../../../components/Button';
 
 const profile = () => {
+  const [selectedImage, setSelectedImage] = useState<any>(null);
```
Comment (elenatrr) [inline app/(tabs)/profile/index.tsx]: I would add a specific type here instead of any, e. g. <string | null>

File: app/(tabs)/profile/index.tsx [hunk 2]
```diff
@@ -1,10 +1,37 @@
...
+  async function pickImageAsync() {
+    try {
+    let result = await ImagePicker.launchImageLibraryAsync({
+      allowsEditing: true,
+      quality: 1,
+    });
+  
+    if (!result.canceled) {
+      setSelectedImage(result.assets[0].uri);
+      console.log(result);
```
Comment (elenatrr) [inline app/(tabs)/profile/index.tsx]: could you remove this console.log, please

File: app/(tabs)/profile/index.tsx [hunk 3]
```diff
@@ -1,10 +1,37 @@
...
+    } else {
+      alert('You did not select any image.');
+    }
+  } catch (error){
+    console.error(error)
```
Comment (elenatrr) [inline app/(tabs)/profile/index.tsx]: let's 'alert' the error instead of logging it

File: app/(tabs)/profile/index.tsx [hunk 4]
```diff
@@ -1,10 +1,37 @@
...
-    <View>
-      <Stack.Screen options={{headerShown: true, title: "Profile"}}/>
+    <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center'}}>
+      <Stack.Screen options={{
+        headerShown: true, 
+        title: "Profile",
+        headerTitleAlign: 'center'
+        }}/>
+        <Button theme="circular" label="Add Profile Photo" onPress={pickImageAsync} />
+        <Button label="Use this photo" />
```
Comment (elenatrr) [inline app/(tabs)/profile/index.tsx]: dont need this button

**Review by elenatrr — CHANGES_REQUESTED** [review]

**Review by elenatrr — APPROVED** [review]

**Pull request merged**